3rdparty/java/BUILD

```diff
@@ -6,90 +6,90 @@ java_library(
     name = "aws-java-sdk",
```

Comment:
Hi @kramasamy I have fixed this bug. 

The Travis CI still not pass yet, I will  get into the root of the matter to find out why.